wagtail_localize/tests/test_submit_translations.py

```diff
@@ -807,6 +804,14 @@ def test_post_submit_snippet_translation(self):
         translated_snippet = self.en_snippet.get_translation(self.fr_locale)
         self.assertEqual(translated_snippet.field, "Test snippet")
 
+        self.assertRedirects(
+            response,
+            reverse(
+                "wagtailsnippets:edit",
+                args=["wagtail_localize_test", "testsnippet", translated_snippet.id],
+            ),
+        )
+
```

Comment:
suggestion: add a test to check that submitting a translation to 2+ locales takes you to the previous locations

Reply:
These two tests already cover that scenario so I don't think we need to add a new test for this.

* `test_post_submit_page_translation_into_multiple_locales`
* `test_post_submit_snippet_translation_into_multiple_locales`